src/test/llmnode_test.cpp

```diff
@@ -87,6 +96,74 @@ class LLMFlowHttpTest : public ::testing::Test {
         while ((server.getModuleState(ovms::SERVABLE_MANAGER_MODULE_NAME) != ovms::ModuleState::INITIALIZED) &&
                (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::high_resolution_clock::now() - start).count() < numberOfRetries)) {
         }
+
+        try {
+            plugin_config_t tokenizerPluginConfig = {};
+            std::string device = "CPU";
+            ov::genai::SchedulerConfig schedulerConfig = {
+                .max_num_batched_tokens = 256,
+                .cache_size = 8,
```

Comment:
not sure we need this much for this kind of testing
if there's only one request being processed at a time let's change it to 1